04-onion-routing.md

```diff
@@ -339,6 +347,10 @@ The reader:
       - incoming `amount_msat` < `amt_to_forward`.
```

Comment:
A few lines above (old line 322: "MUST return an error if the payload contains other tlv fields than"): we need to add `upgrade_accountability` to the list of TLVs allowed.